bitnami/kafka/templates/_helpers.tpl

```diff
@@ -80,3 +80,39 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- $name := default "zookeeper" .Values.zookeeper.nameOverride -}}
 {{- printf "%s-%s" .Release.Name $name | trunc 24 | trimSuffix "-" -}}
 {{- end -}}
+
+{{/*
+Return the proper Docker Image Registry Secret Names
+*/}}
+{{- define "kafka.imagePullSecrets" -}}
+{{/*
+Helm 2.11 supports the assignment of a value to a variable defined in a different scope,
+but Helm 2.9 and 2.10 does not support it, so we need to implement this if-else logic.
+Also, we can not use a single if because lazy evaluation is not an option
+*/}}
+{{- if .Values.global }}
+{{- if .Values.global.imagePullSecrets }}
+imagePullSecrets:
+{{- range
+.Values.global.imagePullSecrets }}
+  - name: {{ . }}
+{{- end }}
+{{- else if or .Values.image.pullSecrets .Values.metrics.image.pullSecrets }}
```

Comment:
I see that `.Values.metrics.image.pullSecrets` does not exists for kafka because it has two ways ox exporting metrics.

Instead, the keys that exists are `.Values.metrics.kafka.image.pullSecrets` and `.Values.metrics.jmx.image.pullSecrets`